### PR TITLE
Removes npm-check-updates from package.json

### PR DESCRIPTION
**Pull request by vegaro**
Should also fix https://github.com/RevenueCat/react-native-purchases/network/alert/example/yarn.lock/dot-prop/open

### CODE REVIEW

File: example/package.json [hunk 1]
```diff
@@ -20,7 +20,6 @@
     "eslint-plugin-react": "^7.14.3",
     "get-dev-paths": "^0.1.1",
     "metro-config": "^0.56.0",
-    "npm-check-updates": "^3.1.21",
```
Comment (aboedo) [inline example/package.json:23]: LOL this line triggered 3.5k additions and 5.4k removals. gotta love npm. 
I'm a bit confused as to why this fixes the dot-prop dependency, though. was it a dependency of `npm-check-updates`?

Comment (vegaro) [reply example/package.json:23]: I know haha. Yes, `dot-prop` was a dependency of `npm-check-updates`

**Comment by vegaro** [comment]
I have no idea why the iOS tests are failing. I will look into it in another moment

File: example/yarn.lock [hunk 2]
```diff
@@ -6852,55 +5573,45 @@ react-devtools-core@^3.6.1:
...
+react-is@^16.7.0, react-is@^16.8.1, react-is@^16.8.4, react-is@^16.8.6, react-is@^16.9.0:
+  version "16.13.1"
+  resolved "https://registry.yarnpkg.com/react-is/-/react-is-16.13.1.tgz#789729a4dc36de2999dc156dd6c1d9c18cea56a4"
+  integrity sha512-24e6ynE2H+OKt4kqsOvNd8kBpV65zoxbA4BVsEOB3ARVWQki/DHzaUoC5KuON/BiccDaCCTZBuOcfZs70kR8bQ==
 
 react-native-gesture-handler@^1.4.0:
-  version "1.4.0"
-  resolved "https://registry.yarnpkg.com/react-native-gesture-handler/-/react-native-gesture-handler-1.4.0.tgz#115f9b0a8f79ca96334f876a67c0122501ef5081"
-  integrity sha512-dbsDWU1mqNrKjsPs1t2CGnRUaGF7U/GN2JbH1AejkC/Wg6WbzGuxZaCPU9yotlXefZpLNQVemmU1bhMTo4IiWw==
+  version "1.7.0"
```
Comment (vegaro) [inline example/yarn.lock:5591]: This update is probably breaking the tests

**Comment by vegaro** [comment]
yes! looks like downgrading that dependency fixed it 🤷 

One day we should work on bringing the latest updates of the dependencies, just to keep the sample up to date

**Pull request merged**